deebot_client/hardware/deebot/xco2fc.py

```
@@ -0,0 +1 @@
+p1jij8.py
```

Ruff (0.8.0)

Found useless expression. Either assign it to a variable or remove it.

(B018)